### PR TITLE
Fixed the bug of throwing 404 if no version exists.

### PR DESCRIPTION
**Pull request by sima-zhu**
Fixed issue: https://github.com/Azure/azure-sdk/issues/1668

Currently solution: if the open page is not 200, jump to the sdk api/index.html page.

### CODE REVIEW

File: doc/ApiDocGeneration/templates/matthews/styles/main.js [hunk 1]
```diff
@@ -118,9 +133,12 @@ function populateOptions(selector, packageName) {
 
         $(versionselector).change(function () {
             targetVersion = $(this).val()
+            original
```
Comment (weshaggard) [inline doc/ApiDocGeneration/templates/matthews/styles/main.js:136]: What does this line do?

**Review by weshaggard — APPROVED** [review]
I don't know that I fully understand this code but beyond the one question it seems reasonable. Please just please be sure to test it.

File: doc/ApiDocGeneration/templates/matthews/styles/main.js (hunk 1 above)
Comment (sima-zhu) [reply doc/ApiDocGeneration/templates/matthews/styles/main.js:136]: This is added by accident. I will remove it. The code tested in local. There is no good choice to test in prod

**Comment by jsquire** [comment]
Hi @sima-zhu.  There hasn't been recent engagement on this PR.  Would you please be so kind as to let us know if this is still an active work stream or if the PR should be closed out?

**Comment by sima-zhu** [comment]
@jsquire Thanks for reminding! It is out of date PR. Close it.

**Pull request closed without merging**